README.md

```diff
@@ -1,194 +1,122 @@
-- [Install](#install)
-- [Usage](#usage)
-- [Configuration](#configuration)
-- [Metric reporting](#metric-reporting)
-- [Security](#security)
-- [Support](#support)
-- [Maintainers](#maintainers)
-- [Contributing](#contributing)
-- [Licensing](#licensing)
-- [License](#license)
+OneSDK is a universal API client and core library of Superface. It provides a simple and uniform interface across many APIs and automatic runtime switching between API providers.
 
-## Background
+See [how Superface works](https://superface.ai/how-it-works) and [get started](https://superface.ai/docs/getting-started).
 
-Superface (super-interface) is a higher-order API, an abstraction on top of the modern APIs like GraphQL and REST. Superface is one interface to discover, connect, and query any capabilities available via conventional APIs. 
+## Important Links
 
-Through its focus on application-level semantics, Superface decouples the clients from servers, enabling fully autonomous evolution. As such it minimizes the code base as well as errors and downtimes while providing unmatched resiliency and redundancy. 
-
-Superface allows for switching capability providers without development at a runtime in milliseconds. Furthermore, Superface decentralizes the composition and aggregation, and thus creates an Autonomous Integration Mesh.
-
-Motivation behind Superface is nicely described in this [video](https://www.youtube.com/watch?v=BCvq3NXFb94) from APIdays conference.
-
-You can get more information at https://superface.ai and https://superface.ai/docs.
+- [Superface website](https://superface.ai)
+- [Get Started](https://superface.ai/docs/getting-started)
+- [Documentation](https://superface.ai/docs)
+- [Discord](https://sfc.is/discord)
 
 ## Install
 
-To install the package, run in the project directory:
+Install the project into Node.js project with npm:
 
-```
-# npm users
+```shell
 npm install @superfaceai/one-sdk
-# yarn users
-yarn add @superfaceai/one-sdk
 ```
 
-## Usage
-
-### Using the OneSDK
-
-To interact with Superface, initialize a new Superface OneSDK instance, references the profile and use case you're wanting to use, then perform it to get the result.
+or Yarn:
 
-#### Initializing the OneSDK client
-
-```typescript
-import { SuperfaceClient } from '@superface/one-sdk';
-
-const client = new SuperfaceClient();
+```shell
+yarn add @superfaceai/one-sdk
 ```
 
-#### Performing the use case
+## Usage
 
-**Note**: You can change url of API requests by setting `SUPERFACE_API_URL` environment variable to desired base url.
+💡 **For quick usage example, check [get started](https://superface.ai/docs/getting-started) documentation.**
 
-Make sure a profile is installed by running `superface install <profileName>[@<profileVersion>]` in the project directory, then load the profile:
+OneSDK works with [Superface CLI](https://github.com/superfaceai/cli). First you install a profile for the use-case you want to use by executing this command in the project directory:
```

Comment:
"With OneSDK everything revolves about your application's use cases for an API. To get started, first install a use case profile using the superface CLI ..."